src/components/v5/common/ActionSidebar/partials/GroupedActionList/GroupedActionItem.tsx

```diff
@@ -0,0 +1,61 @@
+import { type Icon } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+
+import { type Action } from '~constants/actions.ts';
+import { useActionSidebarContext } from '~context/ActionSidebarContext/ActionSidebarContext.ts';
+import { formatText } from '~utils/intl.ts';
+import PillsBase from '~v5/common/Pills/PillsBase.tsx';
+
+import { ACTION_TYPE_FIELD_NAME } from '../../consts.ts';
+
+type ThemeColor = 'blue' | 'success' | 'purple' | 'orange';
+
+export interface GroupedActionItemProps {
+  title: string;
+  description: string;
+  Icon: Icon;
+  action: Action;
+  color?: ThemeColor;
+  isNew?: boolean;
+}
+
+export const GroupedActionItem: FC<GroupedActionItemProps> = ({
+  title,
+  description,
+  Icon,
+  action,
+  color = 'blue',
+  isNew = false,
+}) => {
+  const {
+    actionSidebarToggle: [, { toggleOn: toggleActionSidebarOn }],
+  } = useActionSidebarContext();
+
+  const onClick = () => {
+    toggleActionSidebarOn({
+      [ACTION_TYPE_FIELD_NAME]: action,
+    });
+  };
+  const iconColorClass = `text-${color}-400`;
```

Comment:
this is one thing I'm not a fan of with Tailwind, buuuut https://tailwindcss.com/docs/content-configuration#dynamic-class-names they discourage this type of classNames. 
This may be a larger discussion, since a ternary won't work here